src/extra/libs/rlottie/lv_rlottie.c

```diff
@@ -217,15 +221,18 @@ static void next_frame_task_cb(lv_timer_t * t)
     lv_rlottie_t * rlottie = (lv_rlottie_t *) obj;
 
     if((rlottie->play_ctrl & lv_rlottie_pause) == lv_rlottie_pause) {
-        if(rlottie->current_frame == rlottie->dest_frame)
```

Comment:
We pause the animation to display the current frame statically, and may not need to be reset to the destination frame.

Reply:
Hi Xao Wang, 

  The logic here is if one is setting a destination frame and then call pause, it'll be displayed **and then** the animation (and timer) will pause. You are reviewing an outdated code (or I changed it while you wrote your message).

It's better this way because else, if you pause the animation while it hasn't even started (like in your constructor), it'll not display anything. 
In my code, it'll display **one** frame exactly and then pause.

Reply:
Ok, I get it. You want to call pause without changing the current frame index in the animation.

Reply:
Should be better now. Let me know what you think.

Reply:
It's better now, this is what i want.
Also, i have an idea, i want to refresh a frame after creating lottie, that is to say, call next_frame_task_cb in the constructor.

Reply:
The code will actually do that, see:
1. You create lottie.
2. (task is created in lv_rlottie_constructor)
3.  You call set_current_frame (`current_frame` is updated)
4. Optionally you change the mode to pause
4. (timer is run, since `dest_frame` is not `current_frame`, the frame will be rendered)
5. (timer is run again, but now, `dest_frame` is `current_frame`, so the timer is paused)

If you don't change the mode to pause, it'll run from the frame you've set.

Reply:
I want execute `next_frame_task_cb ` once synchronously in the constructor, so that it can be updated immediately.

Reply:
Even if you execute `next_frame_task_cb`, it won't be updated immediately. The image is only *invalidated* in the timer callback, it'll only be drawn at the rendering time which will happen after the timer is run anyway, if I understand correctly. So, in reality it won't change anything except for making the code more complex.

Reply:
If we call `next_frame_task_cb `, rlottie's image source is confirmed, then we can call `lv_refr_now` to refresh the screen.

Reply:
I'm really not fond of object triggering a full screen refresh when constructed. If the later objects in your function takes longer to construct you'll see an incomplete screen. If you want to only display the first frame, use this code, it'll work with almost no overhead:
```
lv_obj_t * rlottie = lv_rlottie_create_from_file(obj, 128, 128, "test.json");
lv_rlottie_set_play_mode(rlottie, LV_RLOTTIE_PAUSE); 
```